script.js

```diff
@@ -12,3 +12,253 @@ mobileLinks.addEventListener('click', () => {
   title.style.display = 'block';
   mobileLinks.classList.toggle('open');
 });
+
+function createLinks(array) {
+  let list = '';
+  array.forEach((item) => {
+    list += `<li> <a  class="is-link" href="#">${item}</a></li>`;
+  });
+  return list;
+}
+// function createLinks(array) {
+//   const list = document.createElement("ul");
+//   array.forEach((item) => {
+//     list.innerHTML = `<li> <a  class="is-link" href="#">${item}</a></li>`;
+//   });
+//   return list;
+// }
```

Comment:
- [ ] Please remove all commented code from your codebase as this does not make the code neat.